.travis.yml

```diff
@@ -2,3 +2,9 @@ language: ruby
 rvm:
   - 2.1.0
   - 2.2.0
+  - 2.0.0
+  - 1.9.3
+  - ruby-head
+gemfile:
+  - Gemfile
+  - gemfiles/rspec2.gemfile
```

Comment:
Multiple Gemfile Support!


  - On Travis CI, test against RSpec 2 and 3
    - http://docs.travis-ci.com/user/languages/ruby/#Testing-against-multiple-versions-of-dependencies
  - Locally, you can manually test against RSpec 2 and 3
    - RSpec 3
      - `bundle install`
      - `bundle exec rake`
    - RSpec 2
      - `BUNDLE_GEMFILE=gemfiles/rspec2.gemfile bundle install`
      - `BUNDLE_GEMFILE=gemfiles/rspec2.gemfile bundle exec rake`